### DISCUSSION TITLE
Already (slightly) confused

### DISCUSSION THREAD

**Discussion by 1DontEx1st** [aderusha/DDD-Printable-Wall-Control-System#13] [Q&A] [answered]
Just found this tonight and I already know that I'm going to go through a lot of PLA. The first thing I'm attempting to print is the 5x1 Porter Cable charger holder. I already started printing the 5x1 piece, and 1 each of 5x0 Flat Right/Left. I was thinking "just need two sidepieces and one centerpiece", but it looks like I'm missing something (or something_s_). Do I need to print out (6?) pins to attach this all together, or am I even more confused than I think I am?

Thanks,
Darren

**Comment by aderusha** [comment, marked as answer]
Hey no worries my friend!  The pins are in fact what you're missing.  Here's your recipe:

* 1x [5x1 Porter Cable PCXMVC Charger.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/5x1%20Porter%20Cable%20PCXMVC%20Charger.stl)
* 1x each 5x0 Flat [Left](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/5x0%20Flat%20Left.stl) and [Right](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/5x0%20Flat%20Right.stl)
* 6x [4x10x8mm Pin.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/4x10x8mm%20Pin.stl)
* 2x [8mm Lock Pin.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/8mm%20Lock%20Pin.stl)

**Reply by 1DontEx1st** [reply]
Awesome, thanks! (created a [PR](https://github.com/aderusha/DDD-Printable-Wall-Control-System/pull/14) to add this)

**Reply by tweakrr** [reply]
Thanks, yeah perhaps adding a few examples like this to the general readme would be helpful

**Reply by tweakrr** [reply]
Also, is there a guide/rule/best practice for when to create centerpieces that have their own tabs vs creating centerpieces that require separate pins?

For example, [vampliers](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/1x2%20Vampliers%20VT-001-8.stl) has integrated tabs, and it seems most things have the tabs, but some others don't like the OP's question about the [5x1 Porter Cable PCXMVC Charger.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/5x1%20Porter%20Cable%20PCXMVC%20Charger.stl)

**Reply by aderusha** [reply]
That is always the result of print orientation.  If the model can be printed with the tabs on the bed, then tabs are used.  If they would be floating in space, requiring supports, then the inserts are used instead.

I fricken hate supports!

**Reply by tweakrr** [reply]
Ah, that makes a lot of sense! Thanks for clarifying
